.eslintignore

```diff
@@ -3,3 +3,4 @@
 .eslintignore
 pnpm-lock.yaml
 pnpm-workspace.yaml
+packages/stg-evm-usdt-v2/**
```

Comment:
This was originally added because the contracts from the tether repo were not following linting rules and I was hesitant to change anything, but I will remove this and see if the build is still failing